src/entities/Erc1155Token/Erc1155Token.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable camelcase */
+import { TokenInfo } from '@airswap/typescript';
+
+export interface Erc1155TokenAttribute {
+  trait_type: string;
```

Comment:
```suggestion
  'trait_type': string;
```

Reply:
Don't circumvent rules by doing tricks 😛 . Either changes the rule or adhere to it.

Reply:
I know, but I had no strong case for changing it :rofl: